### DISCUSSION TITLE
plotntf error

### DISCUSSION THREAD

**Discussion by cpiga** [Chia-Network/chia-blockchain#8469] [Support]
I have full node and wallet synced. But if I ask the pool of my plotntf file I don't see any partial...
With "chia plotntf show" I get the following response. Please help...
>(venv) chia@chia:~/chia-blockchain$ chia plotnft show
Wallet height: 857294
Sync status: Synced
Wallet id 2:
Current state: FARMING_TO_POOL
Current state from block height: 553020
Launcher ID: 52c25a659c6aacc1d1e51bc5350a14cad2b5ddff7d4551c417bb30b5044e7982
Target address (not for plotting): xch19ukfhgdjx9w5zw5jkhcrf7sr9qkvhgtk0lv6u7c5m9ptj60dt4tscqkmna
Number of plots: 264
Owner public key: b4593f2eb500b938912fb5dda569d789d44cd9d4f44aafc4ac1581f871e5eb6c43ecbe26d42c66750c3d31c1a95a324b
Pool contract address (use ONLY for plotting - do not send money to this address): xch18sl5lmraczx27erm4t5nfuvq6px4h0jme6xl4x3mhjef3w3jjrus4sa24p
Current pool URL: https://eu1.pool.space
Relative lock height: 64 blocks
Exception from 'wallet' <bytes32: 52c25a659c6aacc1d1e51bc5350a14cad2b5ddff7d4551c417bb30b5044e7982>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f7ccfbda5c0>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f7ccfc26528>, 313942.835957473)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f7ccfbda278>

**Comment by cpiga** [comment]
Solved. Missing plot_list parameter in yaml config file. Copied from backup.
Bye Thanks.

**Comment by BrandtH22** [comment]
@cpiga , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
